### PR TITLE
Docs: update sharing

### PR DESCRIPTION
**Pull request by stsewd**
This is to match the changes in .com about supporting per-version
sharing.

### CODE REVIEW

**Comment by stsewd** [comment]
The PR from .com was merged, this is ready for review now.

File: docs/commercial/sharing.rst [hunk 1]
```diff
@@ -10,18 +10,21 @@ You can share your project with users outside of your company:
 * by sending them a *secret link*,
 * by giving them a *password*.
 
-These methods will allow them to view a specific project inside your company.
+These methods will allow them to view specific projects and versions inside your company.
```
Comment (ericholscher) [inline docs/commercial/sharing.rst:13]:
```suggestion
These methods will allow them to view specific projects or versions of a project inside your organization.
```

File: docs/commercial/sharing.rst [hunk 2]
```diff
@@ -10,18 +10,21 @@ You can share your project with users outside of your company:
...
 
 Additionally, you can use a HTTP Authorization Header.
 This is useful to have access from a script.
 
 Enabling
 --------
 
-* Go into your *Project Admin* page and to the *Sharing* menu.
-* Under the *Share with someone new* heading, select the way you prefer (secret link, password, or HTTP header token),
+* Go into your *Project Admin* page and click on :guilabel:`Sharing`.
```
Comment (ericholscher) [inline docs/commercial/sharing.rst:21]:
```suggestion
* Go into your project's :guilabel:`Admin` page and click on :guilabel:`Sharing`.
```

To keep it consistent looking.

**Review by ericholscher — APPROVED** [review]
💯

File: docs/commercial/sharing.rst [hunk 3]
```diff
@@ -10,18 +10,21 @@ You can share your project with users outside of your company:
...
 --------
 
-* Go into your *Project Admin* page and to the *Sharing* menu.
-* Under the *Share with someone new* heading, select the way you prefer (secret link, password, or HTTP header token),
+* Go into your project's :guilabel:`Admin` page and click on :guilabel:`Sharing`.
+* Click on :guilabel:`New Share`
+* Select access type (secret link, password, or HTTP header token),
   add an expiration date and a *Description* so you remember who you're sharing it with.
-* Click *Share!* to create.
+* Check ``Allow access to all versions?`` if you want to grant access to all versions,
```
Comment (humitos) [inline docs/commercial/sharing.rst:25]: Shouldn't we use `:guilabel:` here?

**Pull request merged**